core/src/main/scala/org/apache/spark/shuffle/IndexShuffleBlockResolver.scala

```diff
@@ -58,19 +58,19 @@ private[spark] class IndexShuffleBlockResolver(
     conf: SparkConf,
     // var for testing
     var _blockManager: BlockManager,
-    val taskIdMapsForShuffle: JMap[Int, OpenHashSet[Long]])
+    val taskIdMapsForShuffle: ConcurrentHashMap[Int, OpenHashSet[Long]])
```

Comment:
Can the type be the `ConcurrentMap` interface?

Reply:
nit: I still recommend using the `ConcurrentMap` interface here.

Reply:
Updated.